app/templates/manage_communications.html

```diff
@@ -0,0 +1,106 @@
+{% import 'macros/toolkit_forms.html' as forms %}
+
+{% extends "_base_page.html" %}
+
+{% block content %}
+{%
+    with items = [
+        {
+            "link": url_for('.index'),
+            "label": "Admin home"
+        }
+    ]
+%}
+{% include "toolkit/breadcrumb.html" %}
+{% endwith %}
+
+<div class="page-container">
+{% with messages = get_flashed_messages(with_categories=true) %}
+{% if messages %}
+{% for category, message in messages %}
+{% if category == 'upload_communication' %}
```

Comment:
This should use [the notification banner template](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/notification-banner.html) from the toolkit, rather than being hard-coded HTML.